utils/getRoutes.js

```diff
@@ -11,3 +10,15 @@ export default async () => {
 
   return articles
 }
+
+export const getFeatureRoutes = async (params) => {
+  const {
+    data: { data },
+  } = await axios.get(`https://cms.formester.com/api/features?populate=deep`)
```

Comment:
May confuse the devs. 

I think blow code will be eaiser
res = await ....
res.data

Reply:
we have to make it `res.data.data` in that case.. 
strapi's response: 
```json
{
    "data": [...]
}
```
And axios add another data infornt of this..
```json
{
    "data": {
        "data": [...]
    }
}
```
So `{data: { data }}` destructures it to get the content.